dist/component/component-war/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -27,5 +27,7 @@
       <module name="org.jboss.resteasy.resteasy-jackson2-provider" services="import"/>
       <module name="org.hawkular.bus"/>
     </dependencies>
+    <module name="org.infinispan" export="true" />
+    <module name="org.infinispan.commons" export="true"/>
```

Comment:
Since you have been handling these changes and are more familiar, I will defer to you on what needs to be done. I'd like to get any other changes needed done ASAP so @mwringe can build/test.